js/create_data.js

```diff
@@ -91,62 +118,32 @@ function roundToTwo(num) {
     return +(Math.round(num + "e+2") + "e-2");
 }
 
+// ****
```

Comment:
This is what I meant about using more functions in the general comments; a short function like `addRent` here creates a context in which the `tenant` is whichever tenant is passed to the function - this means the logic can be reused easily without the function itself having to worry about *which* tenant it is modifying.

Reply:
Are you suggesting that I should create addRent method in the tenant object? That could make sense that in the future I'll need a getter and setter to retrieve and save updated dated object in a database.